src/mono/mono/mini/aot-runtime.c

```diff
@@ -4329,7 +4329,6 @@ find_aot_method_in_amodule (MonoAotModule *code_amodule, MonoMethod *method, gui
 	guint32 table_size, entry_size, hash;
 	guint32 *table, *entry;
 	guint32 index;
-	static guint32 n_extra_decodes;
```

Comment:
This is debug code, it should be commented out instead of removed.

Reply:
Commented them instead.